tsconfig.strict.json

```diff
@@ -156,6 +156,7 @@
     "lib/modules/manager/gitlabci-include/index.ts",
     "lib/modules/manager/gitlabci/extract.ts",
     "lib/modules/manager/gitlabci/index.ts",
+    "lib/modules/manager/gitlabci/utils.ts",
```

Comment:
Required because of the new imports in `utils.ts` which are not strictly checked (yet!)